### PR TITLE
add ftrack documentation

### PR DESCRIPTION
**Pull request by 64qam**
Ftrack documentation for a user (Artist, Ftrack actions) and admin (Modules) pages.

### CODE REVIEW

File: website/docs/manager_ftrack_actions.md [hunk 1]
```diff
@@ -124,41 +110,23 @@ This action gives ability to *stop running jobs*. When action is triggered, an i
...
 ![prepare_project-icon](assets/ftrack/ftrack-prepare_project-icon.png)
+
 </div>
 </div>
 
 * Entity types: Project
 * User roles: Pypeclub, Administrator, Project manager
 
-Allows project managers and coordinator to *set basic project attributes* needed for OpenPype to operate, *Create project folders* if you want and especially *prepare Project specific [anatomy](admin_config#anatomy) and [presets](admin_config#presets)* files for you.
+Allows project managers and coordinator to *set basic project attributes* needed for OpenPype to operate, *Create project folders* if you want and especially prepare project specific [anatomy](admin_settings_project_anatomy) or [settings](admin_settings_project).
```
Comment (mkolar) [inline website/docs/manager_ftrack_actions.md:129]: Let's remove the mention of anatomy, that will need to be done in settings. plus it's not actually being copied from 3.0

File: website/docs/manager_ftrack_actions.md [hunk 2]
```diff
@@ -220,47 +194,106 @@ You should use this action if you need to delete Entities or Asset Versions othe
...
+
+![ftrack-delivery-icon](assets/ftrack/ftrack-delivery-icon.png)
+
+</div>
+</div>
+
+* Entity types: Task
+* User roles: Pypeclub, Project manager, Administrator
+
+Collects approved hires files and copy them into a folder. It usually creates h.264 files for preview and mov for editorial. All files are then copied according to predefined naming convention to a specific folder.
```
Comment (mkolar) [inline website/docs/manager_ftrack_actions.md:220]: This doesn't create any review files. It just takes any components of any versions and copies and renames them correctly.

File: website/docs/module_ftrack.md [hunk 3]
```diff
@@ -8,38 +8,33 @@ import Tabs from '@theme/Tabs';
...
+You can only use our Ftrack Actions and publish to Ftrack if each artist is logged in.
 
 
 ### Custom Attributes
-After successfully connecting OpenPype with you ftrack, you can right on any project in ftrack and you should see a bunch of actions available. The most important one is called `OpenPype Admin` and contains multiple options inside.
-
-To prepare ftrack for working with OpenPype you'll need to run [OpenPype Admin - Create/Update Avalon Attributes](manager_ftrack_actions#create-update-avalon-attributes), which creates and sets the Custom Attributes necessary for OpenPype to function. 
-
-
+After successfully connecting OpenPype with you Ftrack, you can right on any project in Ftrack and you should see a bunch of actions available. The most important one is called `OpenPype Admin` and contains multiple options inside.
```
Comment (mkolar) [inline website/docs/module_ftrack.md:27]: typo: * right click

File: website/docs/module_ftrack.md [hunk 4]
```diff
@@ -61,7 +56,7 @@ There are specific launch arguments for event server. With `openpype eventserver
 -   `--ftrack-url "https://yourdomain.ftrackapp.com/"` : Ftrack server URL _(it is not needed to enter if you have set `FTRACK_SERVER` in OpenPype' environments)_
 -   `--ftrack-events-path "//Paths/To/Events/"` : Paths to events folder. May contain multiple paths separated by `;`. _(it is not needed to enter if you have set `FTRACK_EVENTS_PATH` in OpenPype' environments)_
 
-So if you want to use OpenPype's environments then you can launch event server for first time with these arguments `$PYPE_SETUP/pype eventserver --ftrack-user "my.username" --ftrack-api-key "00000aaa-11bb-22cc-33dd-444444eeeee" --store-credentials`. Since that time, if everything was entered correctly, you can launch event server with `$PYPE_SETUP/pype eventserver`.
+So if you want to use OpenPype's environments then you can launch event server for first time with these arguments `$OPENPYPE_SETUP/openpype eventserver --ftrack-user "my.username" --ftrack-api-key "00000aaa-11bb-22cc-33dd-444444eeeee" --store-credentials`. Since that time, if everything was entered correctly, you can launch event server with `$OPENPYPE_SETUP/openpype eventserver`.
```
Comment (mkolar) [inline website/docs/module_ftrack.md:59]: OPENPYPE_SETUP doesn't exist anymore. This needs to point to the executable probably

File: website/docs/module_ftrack.md [hunk 5]
```diff
@@ -163,33 +172,45 @@ This event updates entities on their changes Ftrack. When new entity is created
...
+### Update Hierarchy thumbnails
+
+Push thumbnails from version, up through multiple hierarchy levels
+
+### Update status on task action
 
-### Thumbnails update _(ThumbnailEvents)_
+Change status of next task from `Not started` to `Ready` when previous task is approved.
+
+Multiple detailed rules for next task update can be configured in the presets.
```
Comment (mkolar) [inline website/docs/module_ftrack.md:189]: in the settings , not presets

**Review by mkolar — CHANGES_REQUESTED** [review]

File: website/docs/module_ftrack.md (hunk 4 above)
Comment (64qam) [reply website/docs/module_ftrack.md:59]: provisionally OPENPYPE_SETUP replaced for openpype.exe

**Review by mkolar — APPROVED** [review]

**Pull request merged**